services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from django.db import transaction
+from db.models import Ticket, Order, User, MovieSession

```

Comment:
Replace direct usage of `User` with `get_user_model()` for better practice and flexibility. This aligns with the checklist recommendation to use `get_user_model()` instead of `User`.